frontend/src/component/feature/StrategyTypes/StrategyVariants.tsx

```diff
@@ -0,0 +1,129 @@
+import {
```

Comment:
This entire file is taken from POC and will be rewritten from scratch so no need to review as it's hidden behind a flag. This is to allow testing in sandbox.